lib/Map/LeafletDataSourceDisplay.js

```diff
@@ -196,6 +197,39 @@ LeafletDataSourceDisplay.prototype.update = function(time) {
     return result;
 };
 
+/**
+ * Computes the rectangular bounds which encloses the visualization
+ * produced for the given entities.
+ *
+ * @param {EntityCollection} entities The collection of entities whose bounds to compute.
+ * @returns {LatLngBounds} The computed bounds.
+ */
+LeafletDataSourceDisplay.prototype.getLatLngBounds = function(entities) {
```

Comment:
The caller of this function already has a data source (in the `target` variable), so if you passed that into this function instead of `target.entities`, you wouldn't need to find the data source below. You could skip down to `var visualizers = dataSource._visualizersByDisplayID[this._displayID]`.